stall_test/stall_test.ino

```diff
@@ -0,0 +1,119 @@
+#include "Arduino.h"
+#include "Motors.h"
+#include <Encoder.h>
+#include <ros.h>
+#include <std_msgs/Float32.h>
+
+#define LEFT_ENCODER_PIN1 34 // TODO: Why are there duplicate #defines from the other .ino file?
```

Comment:
remove todo comments if already fixed.
remove all todo comments if already fixed or else leaves posibility of confusion in the future.
address this issue for all files

Reply:
The stall test folder was just the test code Kevin wrote before integrating with ians_controller. The reason it has duplicate defines is because it's a whole separate project. It can be deleted